browser/resources/settings/brave_routes.ts

```diff
@@ -88,9 +88,10 @@ export default function addBraveRoutes(r: Partial<SettingsRoutes>) {
     r.FONTS = r.BRAVE_CONTENT.createChild('/fonts')
     r.FONTS.hasMigratedToPlugin = true
   }
-  if (pageVisibility.surveyPanelist) {
+  if (pageVisibility.surveyPanelist && r.PRIVACY) {
     r.BRAVE_SURVEY_PANELIST =
-      r.BASIC.createSection('/surveyPanelist', 'surveyPanelist')
+      r.PRIVACY.createChild('/surveyPanelist', 'surveyPanelist')
```

Comment:
the problem for direct navigation was that this wasn't a child of the PRIVACY route

Reply:
Oh, I absolutely called `createSection`. Argh!!!